Frontend-CB/src/assets/Navbar.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const Navbar: React.FC = () => {
+  return(
+    <nav>
+      {<li><a href="/">Main</a></li>
+      <li><a href="/about">App</a></li>
+      <li><a href="/services">Login</a></li>
+      }
+    </nav>
```

Comment:
Have you took a look at the [figma prototype] for making the nav bar?(https://www.figma.com/proto/QoeXN2FtKyzmbUgcINgyiU/Commerce-Bank-Prototype?node-id=1-2&starting-point-node-id=1%3A2&mode=design&t=JXT1CVUPVMzKfFNf-1)